lib/tucan.ex

```diff
@@ -4265,10 +4265,15 @@ defmodule Tucan do
   This sets both width and height at once.
   """
   @doc section: :styling
-  @spec set_size(vl :: VegaLite.t(), width :: pos_integer(), height :: pos_integer()) ::
+  @spec set_size(
+          vl :: VegaLite.t(),
+          width :: pos_integer() | String.t(),
+          height :: pos_integer() | String.t()
```

Comment:
```suggestion
          width :: pos_integer() | :container,
          height :: pos_integer() | :container
```

Should we make it an atom instead? VegaLite will make it a string in the spec.

Reply:
Cool!